_docs/results/duckduckbot.md

```diff
@@ -9,10 +9,17 @@ order: 1400
 ## What Is DuckDuckBot and What Does It Do?
 
 DuckDuckBot is the Web crawler for <a href="https://duckduckgo.com">DuckDuckGo</a>.
-DuckDuckBot’s job is to constantly improve our search results and offer users the best and most secure search experience possible. The crawler's user agent will appear as `DuckDuckBot-Https/1.1; (+https://duckduckgo.com/duckduckbot)`. It respects <a href="https://metacpan.org/module/WWW::RobotRules">WWW::RobotRules</a> and originates from these IP addresses:
+DuckDuckBot’s job is to constantly improve our search results and offer users the best and most secure search experience possible. The crawler's user agent will appear as `DuckDuckBot-Https/1.1; (+https://duckduckgo.com/duckduckbot.html)`. It respects <a href="https://metacpan.org/module/WWW::RobotRules">WWW::RobotRules</a> and originates from these IP addresses:
```

Comment:
The default UA we use is http (https://dub.duckduckgo.com/duckduckgo/ddg/blob/bttf/lib/DDG/Util/Crawl.pm#L143)  which is `DuckDuckBot/1.1; (+http://duckduckgo.com/duckduckbot.html)`. That's what favicons is updating to as well.